charts/outdated/values.yaml

```
@@ -30,6 +30,7 @@ ember:
 
 django:
   allowedHosts: "*"
+  watchmanChecks: ~
```

Using an array might be cleaner given that Helm will consider that to be falsey as well.
```suggestion
  watchmanChecks: []
```

---

oops